src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,43 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double bladeWidth;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. It's a good practice to encapsulate the fields of a class. Consider using private access modifiers for `bladeWidth` and `enginePower`.